src/maintenance/coverage_crossref_updater.py

```diff
@@ -0,0 +1,44 @@
+"""Updates the table for the `coverage_crossref` endpoint."""
+
+# standard library
+import argparse
+import sys
+import time
+
+# first party
+from delphi.epidata.acquisition.covidcast.database import Database
+from delphi_utils import get_structured_logger
+from delphi.epidata.client.delphi_epidata import Epidata
+
+
+def main():
+  """Updates the table for the `coverage_crossref`."""
+
+  logger = get_structured_logger("coverage_crossref_updater")
+  start_time = time.time()
+  database = Database()
+  database.connect()
+
+  # compute and update coverage_crossref
+  try:
+    coverage = database.compute_coverage_crossref()
+  except:
+    # clean up before failing
+    database.disconnect(True)
+    raise
```

Comment:
this should close the db connection whether it fails or succeeds, and will reraise any exception encountered:
```suggestion
  finally:
    # clean up in success and in failure
    database.disconnect(True)
```
https://docs.python.org/3/tutorial/errors.html#defining-clean-up-actions